src/components/roles/RoleForm.tsx

```
@@ -1,90 +1,172 @@
+        : [...prev.permissionIds, permissionId],
+    }));
   };
 
   return (
-    <form onSubmit={handleSubmit}>
-      <div className="mb-4">
-        <label className="block text-gray-700 mb-1">Role name</label>
+    <form onSubmit={handleSubmit} className="space-y-6">
+      {/* Role Name */}
+      <div>
+        <label className="block text-sm font-medium text-gray-700 mb-2">Role Name *</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)